src/components/staking/LiValidator.vue

```diff
@@ -77,6 +89,23 @@ export default {
       default: () => "returns"
     }
   },
+  computed: {
+    status() {
+      if (
+        this.validator.jailed ||
+        this.validator.tombstoned ||
+        this.validator.status === 0
+      )
+        return `Inactive`
+      return `Active`
+    },
+    status_detailed() {
+      if (this.validator.jailed) return `Temporally banned from the network`
```

Comment:
@faboweb we need to find a solution for spelling errors. perhaps we can make a PR for this Github app to check HTML.

https://github.com/apps/prosebot

Reply:
Please take care of that